containers/FishCaught.js

```diff
@@ -8,7 +8,7 @@ import {
   TouchableOpacity,
 } from "react-native";
 
-export default Fish = ({ navigation }) => {
+export default function FishCaught ({ navigation }) {
```

Comment:
Thanks for fixing this typo I had. It's interesting that this still was rendering properly on my Expo app on my phone when it shouldn't have due to that naming typo.